addOns/graphql/src/main/javahelp/org/zaproxy/addon/graphql/resources/help/contents/graphql.html

```diff
@@ -56,6 +56,13 @@ <h3> Actions </h3>
 
 The definitions will be imported synchronously and any warnings will be returned.
 
+<h3>GraphiQL IDE</h3>
+A customized version of the <a href="https://github.com/graphql/graphiql">GraphiQL IDE</a> can be accessed via the API.
```

Comment:
```suggestion
A customized version of the <a href="https://github.com/graphql/graphiql">GraphiQL IDE</a> can be accessed via the ZAP API server.
```

Or something like that?